mw-webapp/src/component/editableTextarea/editableTextarea.css

```diff
@@ -0,0 +1,27 @@
+.grow-wrap {
+    display: grid;
+    max-width: 300px;
+}
+
+.grow-wrap::after {
+    content: attr(data-replicated-value) " ";
+    white-space: pre-wrap;
+    visibility: hidden;
+}
+
+.grow-wrap > textarea {
+    resize: none;
+    overflow: hidden;
+    width: '100%';
+    height: '100%';
+    white-space: 'pre-wrap';
+}
+
+.grow-wrap > textarea,
+.grow-wrap::after {
+    border: 1px solid black;
+    padding: 0.5rem;
+    font: inherit;
+    grid-area: 1 / 1 / 2 / 2;
+}
+ 
```

Comment:
Could you attach few images with examples how does it look for now, please?

Reply:
![Uploading span.png…]()

Reply:
![Screenshot_1](https://github.com/tritonJS826/masters-way/assets/85361159/bf73ac18-5099-4cce-8240-800aeb3f5ac9)

Reply:
![Uploading Screenshot_5.png…]()

Reply:
![Uploading Screenshot_4.png…]()

Reply:
![Screenshot_4](https://github.com/tritonJS826/masters-way/assets/85361159/4bfec6af-2661-4916-88a8-9eac4cb05b28)